pulsar-transformations/src/main/java/com/datastax/oss/pulsar/functions/transforms/TransformFunction.java

```diff
@@ -290,6 +295,24 @@ public static FlattenStep newFlattenFunction(JsonNode node) {
     return builder.build();
   }
 
+  private static TransformStep newComputeFieldFunction(JsonNode node) {
+    List<ComputeField> fieldList = new ArrayList<>();
+    node.get("fields")
+        .iterator()
+        .forEachRemaining(
+            it ->
+                fieldList.add(
+                    ComputeField.builder()
+                        .name(it.get("name").asText())
+                        .expression(it.get("expression").asText())
+                        .type(ComputeFieldType.valueOf(it.get("type").asText()))
+                        .part(it.get("part") == null ? null : it.get("part").asText())
+                        .optional(
+                            it.get("optional") == null ? false : it.get("optional").asBoolean())
```

Comment:
I would set it "optional" by default, it is easier, because if it is optional you don't care about the fact that you have nulls or not